examples/kubernetes-ingress/scripts/08-guestbook.sh

```diff
@@ -6,6 +6,8 @@ source "${dir}/helpers.bash"
 
```

Comment:
Add logs using tests/helpers.bash:log so users can follow what's going on in case they hit an issue while the script runs.